app/audio-player-data/src/main/kotlin/com/hedvig/audio/player/data/SIgnedAudioUrl.kt

```diff
@@ -1,9 +1,13 @@
-package com.hedvig.android.audio.player
+package com.hedvig.audio.player.data
+
+import kotlinx.serialization.Serializable
 
 /**
  * A class to hold the url coming from the backend, with an overridden equals in order to return true in case we're
  * dealing with the same audio file but with a different query string
  */
+
+@Serializable
 class SignedAudioUrl private constructor(
```

Comment:
This was not @ Serializable before, I assume it is now because a new destination needs to access it right?

Reply:
Yep, after I merge this I will need to use it somewhere where it has to be serializable, so I've decided to annotate it right away